index.js

```diff
@@ -34,128 +36,154 @@ if (staging) {
   };
 }
 
-app.launch(function(req, res) {
-  res.say(prompt).reprompt(reprompt).shouldEndSession(false);
+app.launch(function (req, res) {
+  res.say(prompt)
+    .reprompt(reprompt)
+    .shouldEndSession(false);
 });
 
 app.intent('GetEstimate', {
-  'slots': slots,
-  'utterances': utterances
-},
-  function(req, res) {
-    var slots = req.data.request.intent.slots;
-    console.log('slots:', slots);
-    var userId = req.userId; // the unique alexa session userId
-    var mode = (staging) ? 'cheapest' : req.slot('MODE'); // cheapest or fastest
-
-    // find the ORIGIN slot that is populated in this request, if any
-    var originArray = _.filter(slots, function(slotValue, slotKey) {
-      return (slotValue.value && slotValue.value.length > 0 && slotKey.includes('ORIGIN'));
-    });
-    var origin = (originArray.length) ? originArray[0].value : null;
-    console.log('Alexa thinks my origin is', origin);
-
-    // find the DESTINATION slot that is populated in this request
-    var destination = _.filter(slots, function(slotValue, slotKey) {
-      return (slotValue.value && slotValue.value.length > 0 && slotKey.includes('DESTINATION'));
-    })[0].value;
-    console.log('Alexa thinks my destination is', destination);
-
-    // todo: grab user location?
-    if (_.isEmpty(mode) || _.isEmpty(destination)) {
-      prompt = 'I didn\'t catch that. Please try again';
-      res.say(prompt).reprompt(reprompt).shouldEndSession(false);
-    } else {
-      var originDescrip, destDescrip, originCoords, destCoords;
-
-      if (origin) {
-        // get originDescrip and originCoords for origin that was passed in
-        rideHelper.placesCall(origin, function(descrip, coords) {
-          originDescrip = descrip;
-          originCoords = coords;
-          if (destDescrip) {
-            // make getEstimate call since destDescrip async call resolved first
-            rideHelper.getEstimate(mode, originCoords, destCoords, function(winner) {
-              var answer = formatAnswer(winner, mode, originDescrip, destDescrip);
-              res.say(answer).send();
+      'slots': {
+        'MODE': 'MODE',
+        'DESTINATION': 'DESTINATION',
+        'DESTINATION_ONE': 'DESTINATION_ONE',
+      },
+      'utterances': [
+        '{Find|Get|Order|Call|Book} {a|one|the|me the|me a} {MODE} {car|ride} to {DESTINATION}'
+      ]
```

Comment:
i think this is old code that you probably chose to keep when you were resolving merge conflicts, but we need the version that assigns slots and utterances properties to the slots and utterances variables that were declared at the top (and configured based on whether staging is true or not)